src/main/resources/static/css/govuk-frontend-4.7.0.min.css

```diff
@@ -0,0 +1,3 @@
+@charset "UTF-8";:root{--govuk-frontend-version:"4.7.0"}.govuk-link{font-family:GDS Transport,arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;text-decoration:underline}
```

Comment:
latest version downloaded using script in auth - amended not to add in the `/auth/` prefix too so that fonts are actually loaded correctly too